src/test/java/LOGIN.java

```diff
@@ -0,0 +1,34 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+import static org.bouncycastle.cms.RecipientId.password;
+
+public class LOGIN extends BASE {
+
+
+    public LOGIN(WebDriver givenDriver) {
+        super(givenDriver);
+    }
+    By emailField = By.cssSelector("input[type='email']");
+
+    By passwordField = By.cssSelector("input[type='password']");
+
+    By submitButton = By.cssSelector("button[type='submit']");
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```